WebHostLib/templates/weightedOptions/weightedOptions.html

```diff
@@ -0,0 +1,119 @@
+{% extends 'pageWrapper.html' %}
+{% import 'weightedOptions/macros.html' as inputs %}
+
+{% block head %}
+    <title>{{ world_name }} Weighted Options</title>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename="styles/markdown.css") }}" />
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename="styles/weightedOptions/weightedOptions.css") }}" />
+    <script type="application/ecmascript" src="{{ url_for('static', filename="assets/weightedOptions.js") }}"></script>
+    <noscript>
+        <style>
+            .js-required{
+                display: none;
+            }
+        </style>
+    </noscript>
+{% endblock %}
+
+{% block body %}
+    {% include 'header/'+theme+'Header.html' %}
+    <div id="weighted-options" class="markdown" data-game="{{ world_name }}">
+        <noscript>
+            <div class="js-warning-banner">
+                This page has reduced functionality without JavaScript.
+            </div>
+        </noscript>
+
+        <div id="user-message"></div>
+
+        <div id="weighted-options-header">
+            <h1>{{ world_name }}</h1>
+            <h1>Weighted Options</h1>
```

Comment:
These titles blend together, and it's really easy to skip over the explanation of the page because of it. Probably a good idea to put another header just above the name section to differentiate it from the explanations.
```suggestion
            <br />
            <h2>Weighted Options</h2>
```

Reply:
I think I've done all I'm going to stylistically until the PR is accepted. Unless something is broken, the styles will remain as they are.